saleor/graphql/app/types.py

```diff
@@ -233,6 +276,16 @@ def resolve_url(root, info: ResolveInfo):
     def resolve_target(root, _info: ResolveInfo):
         return root.target
 
+    @staticmethod
+    def resolve_mount_name(root: models.AppExtension, _info: ResolveInfo):
+        # Convert lowercase mount value to uppercase constant name
+        return root.mount.upper()
+
+    @staticmethod
+    def resolve_target_name(root: models.AppExtension, _info: ResolveInfo):
+        # Convert lowercase target value to uppercase constant name
+        return root.target.upper()
```

Comment:
Having `.upper()` here means, that we will also need to add it in 3.23. Is it expected?